### PR TITLE
[Load][Vectorized] load opt code by change `replace` and `replace_if_not_null` do not copy value

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

After do the opt of this

unique_table 1000W data load time

before:45s -> after:35s

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/aggregate_functions/aggregate_function_window.h [hunk 1]
```diff
@@ -262,58 +266,31 @@ struct LeadAndLagData {
                 col.insert_default();
```
Comment (BiteTheDDDDt) [inline be/src/vec/aggregate_functions/aggregate_function_window.h:266]: Maybe we can sympfy this function to two case `is_null` or not.

File: be/src/vec/aggregate_functions/aggregate_function_window.h [hunk 2]
```diff
@@ -210,26 +210,30 @@ class WindowFunctionNTile final
 struct Value {
 public:
     bool is_null() const { return _is_null; }
-    StringRef get_value() const { return _value; }
-
     void set_null(bool is_null) { _is_null = is_null; }
-    void set_value(StringRef value) { _value = value; }
+    StringRef get_value() const { return _ptr->get_data_at(_offset); }
+
+    void set_value(StringRef ref) {
```
Comment (BiteTheDDDDt) [inline be/src/vec/aggregate_functions/aggregate_function_window.h:216]: Whether use `StringRef` to represent column_ptr and offset is too tricky?

Comment (HappenLee) [reply be/src/vec/aggregate_functions/aggregate_function_window.h:216]: just keep same with `copy_value` do make sure the interface is same

File: be/src/vec/aggregate_functions/aggregate_function_window.h (hunk 1 above)
Comment (HappenLee) [reply be/src/vec/aggregate_functions/aggregate_function_window.h:266]: just for reduce one time `virtual function call`

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Pull request merged**